City-of-Helsinki/drupal-helfi-etusivu
# UHF-5289 news configs

# News node styling [UHF-5289](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5289)
# Of interest-block [UHF-5255](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5255)

Configs for the news node sidebar blocks and new of interest block

## What was done
* Configured the new of interest block and moved the sidebar blocks to the correct region (design looks wonky for now before the region pr is merged)
* Fixed radioactivity configs

## How to install
* Make sure your frontpage is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the HDBT theme
    * `composer require drupal/hdbt:dev-UHF-5289-news-node-theming`
* Update the Helfi Platform config
    * `composer require drupal/helfi_platform_config:dev-UHF-5289-news-node-theming`
    * run `drush updb`
* Checkout this branch `UHF-5289-news-configs` 
* Run `make drush-cr`

## How to test
* Create a few news, some with and one without any tags
    * Check how the "of interest" -block behaves
    * If the news node doesn't have any tags, there should be the newest news in the block
    * If the news node has a tag in common with some other news, the ones with the same tags should appear
    * If the news node has a tag, but no one else has the same tag, the whole block should be hidden
* Check that the sidebars are now in the second sidebar region (they are wrongfully on the left side at the moment but will be fixed in upcoming region pr)
    * The styling of the most read sidebar block should be the same as the newest. (Title, date and without the photo)
* Test that you're now able to add text, images, list of links, banner and remote video to the news content

## Other PRs
* https://github.com/City-of-Helsinki/drupal-hdbt/pull/313
* https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/320
